content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -109,10 +109,15 @@ In the above example:
 * The StatefulSet, named `web`, has a Spec that indicates that 3 replicas of the nginx container will be launched in unique Pods.
 * The `volumeClaimTemplates` will provide stable storage using [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) provisioned by a PersistentVolume Provisioner.
 
+The name of a StatefulSet object must be a valid
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```

Comment:
@tengqm , What do you think about wrapping lines 112 - 113 in a note shortcode? This information is hard to find.

Reply:
I don't see a need in wrapping this into a 'note'. Regarding the 'note' shortcode, my take is that it is designed to call out something unexpected such as some limitations in current implementation.